package.json

```diff
@@ -34,10 +34,13 @@
     "prepush": "npm run lint"
   },
   "dependencies": {
+    "immer": "1.5.0",
```

Comment:
[immer](https://github.com/mweststrate/immer) is pretty small, but it greatly simplifies handling immutability in reducers.